rebar.lock

```diff
@@ -22,6 +22,10 @@
   {git,"git://github.com/ninenines/cowlib.git",
        {ref,"d544a494af4dbc810fc9c15eaf5cc050cced1501"}},
   1},
+ {<<"erlang_syslog">>,
+  {git,"https://github.com/heroku/erlang_syslog",
+       {ref,"0cfef5a95150e11f60fee5c35b24decfcc41066e"}},
+  0},
```

Comment:
This seems wrong. When you look at https://github.com/heroku/erlang_syslog/blob/0cfef5a95150e11f60fee5c35b24decfcc41066e/src/syslog.app.src#L1 the app is called `syslog`. The commit seems correct. I'm not sure why rebar does this.